modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,481 @@
+    credentialsName: kni1-master-0-bmc-secret
+  bootMACAddress: aa:aa:aa:aa:ab:03
+  bootMode: UEFI
+  externallyProvisioned: false
+  hardwareProfile: unknown
+  online: true
+EOF
+----
++
+After the inspection is complete, the `BareMetalHost` object is created and available to be provisioned.
+
+. View available `BareMetalHost` objects:
++[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME           STATE                    CONSUMER                   ONLINE   ERROR   AGE
+kni1-master-0  available                ocp-hkw9p-master-0         true             1h10m
+kni1-master-1  externally provisioned   ocp-hkw9p-master-1         true             4h53m
+kni1-master-2  externally provisioned   ocp-hkw9p-master-2         true             4h53m
+kni1-worker-0  provisioned              ocp-hkw9p-worker-0-ktmmx   true             4h53m
+kni1-worker-1  provisioned              ocp-hkw9p-worker-0-l2zmb   true             4h53m
+----
++
+.. Create the new machine using the `new-master-machine.yaml` file:
++
+[source,terminal]
+----
+$ oc apply -f new-master-machine.yaml
+----
+
+
+.. Verify that the new machine has been created:
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api -o wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        PHASE          TYPE        REGION      ZONE         AGE     NODE                           PROVIDERID                              STATE
+clustername-8qw5l-master-1                  Running        m4.xlarge   us-east-1   us-east-1b   3h37m   ip-10-0-154-204.ec2.internal   aws:///us-east-1b/i-096c349b700a19631   running
+clustername-8qw5l-master-2                  Running        m4.xlarge   us-east-1   us-east-1c   3h37m   ip-10-0-164-97.ec2.internal    aws:///us-east-1c/i-02626f1dba9ed5bba   running
+clustername-8qw5l-master-3                  Provisioning   m4.xlarge   us-east-1   us-east-1a   85s     ip-10-0-133-53.ec2.internal    aws:///us-east-1a/i-015b0888fe17bc2c8   running <1>
+clustername-8qw5l-worker-us-east-1a-wbtgd   Running        m4.large    us-east-1   us-east-1a   3h28m   ip-10-0-129-226.ec2.internal   aws:///us-east-1a/i-010ef6279b4662ced   running
+clustername-8qw5l-worker-us-east-1b-lrdxb   Running        m4.large    us-east-1   us-east-1b   3h28m   ip-10-0-144-248.ec2.internal   aws:///us-east-1b/i-0cb45ac45a166173b   running
+clustername-8qw5l-worker-us-east-1c-pkg26   Running        m4.large    us-east-1   us-east-1c   3h28m   ip-10-0-170-181.ec2.internal   aws:///us-east-1c/i-06861c00007751b0a   running
+----
+<1> The new machine, `clustername-8qw5l-master-3` is being created and is ready once the phase changes from `Provisioning` to `Running`.
++
+It might take a few minutes for the new machine to be created. The etcd cluster Operator will automatically sync when the machine or node returns to a healthy state.
+
```

Comment:
$ oc get bmh -n openshift-machine-api
NAME                 STATE                    CONSUMER                        ONLINE   ERROR   AGE
openshift-master-0   externally provisioned   ocp-edge-4zg89-master-0         true             4h48m
openshift-master-1   externally provisioned   ocp-edge-4zg89-master-1         true             4h48m
openshift-master-2   provisioned              ocp-edge-4zg89-master-3         true             47m
openshift-worker-0   provisioned              ocp-edge-4zg89-worker-0-8v2jj   true             4h48m
openshift-worker-1   provisioned              ocp-edge-4zg89-worker-0-jb5z9   true             4h48m
[kni@ocp-edge06 ~]$ oc get machine -n openshift-machine-api
NAME                            PHASE     TYPE   REGION   ZONE   AGE
ocp-edge-4zg89-master-0         Running                          4h49m
ocp-edge-4zg89-master-1         Running                          4h48m
ocp-edge-4zg89-master-3         Running                          25m
ocp-edge-4zg89-worker-0-8v2jj   Running                          4h23m
ocp-edge-4zg89-worker-0-jb5z9   Running                          4h23m
[kni@ocp-edge06 ~]$ oc get nodes
NAME                 STATUS   ROLES    AGE     VERSION
openshift-master-0   Ready    master   4h26m   v1.24.0+9546431
openshift-master-1   Ready    master   4h26m   v1.24.0+9546431
openshift-master-2   Ready    master   12m     v1.24.0+9546431
openshift-worker-0   Ready    worker   3h58m   v1.24.0+9546431
openshift-worker-1   Ready    worker   3h58m   v1.24.0+9546431

Reply:
did you decide not to give this examples of the expected cluster state?

Reply:
I'm glad to do so but formatting of the output text is challenging.....

Reply:
Done

Reply:
It looks like it added not to the right place (see my previous comment)